libsoftwarecontainer/include/softwarecontainer.h

```diff
@@ -95,28 +108,35 @@ class SoftwareContainer :
      */
     ReturnCode resume();
 
-    void addGateway(Gateway *gateway);
-
-    ReturnCode init();
-
-    std::shared_ptr<ContainerAbstractInterface> getContainer();
-
-    std::shared_ptr<FunctionJob> createFunctionJob(const std::function<int()> fun);
-    std::shared_ptr<CommandJob> createCommandJob(const std::string &command);
-
+    /**
+     *
+     */
     ReturnCode bindMount(const std::string &pathOnHost, const std::string &pathInContainer, bool readonly = true);
 
+    /**
+     *
+     */
```

Comment:
Yeah, I thought it made them stand out more from all the commented ones. I could add comments to them, but good comments are sort of out of scope for the commits. Maybe I should get into that good scout feeling though, but these old sins are too annoying sometimes. I'll remove the empty comments instead (for now).